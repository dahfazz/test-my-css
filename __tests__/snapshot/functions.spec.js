var sass = require('node-sass');

it('Buttons are OK', async () => {
  
  var result = sass.renderSync({
    data: `
    @import 'src/functions';
    body {
      background-color: color-contrast(#ccc);
      color: color-contrast(#333);
    }`,
    outputStyle: 'nested'
  });
  expect(result.css.toString()).toMatchSnapshot();
});