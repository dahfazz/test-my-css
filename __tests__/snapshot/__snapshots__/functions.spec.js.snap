// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons are OK 1`] = `
"body {
  background-color: #000;
  color: #fff; }
"
`;
